src/Workspace/index.jsx

```diff
@@ -431,6 +431,10 @@ class Workspace extends React.Component {
   }
 
   handleMenuClick = async (e) => {
+    if (this.state.payModel.all_pay_models[e.key].request_status === 'above limit') {
+      message.error('Selected pay model usage has exceeded its available funding. Please choose another pay model. Contact brhsupport@datacommons.io with questions.');
```

Comment:
really just a design choice but IMO it is better to disable the menu item and provide these explantory text in the tooltip rater than using this temporoay message banner